docs/content/guides/dagster-pipes/aws-lambda.mdx

```diff
@@ -0,0 +1,288 @@
+---
+title: "Integrating AWS Lambda with Dagster Pipes | Dagster Docs"
+description: "Learn to integrate Dagster Pipes with AWS Lambda to launch external code from Dagster assets."
+---
+
+# Integrating Amazon Web Services Lambda with Dagster Pipes
+
+<Note>
+  <strong>Heads up!</strong> This guide focuses on using an out-of-the-box
+  Amazon Web Services Lambda resource. For further customization, use the{" "}
```

Comment:
nit: i found it a little jarring seeing "Amazon Web Services Lambda" typed out instead of just AWS Lambda